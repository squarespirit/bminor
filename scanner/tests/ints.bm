123 0 -0 -54321 007