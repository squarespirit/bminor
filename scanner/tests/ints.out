intlit	123
intlit	0
intlit	-0
intlit	-54321
intlit	007
